--- origsrc/cantor-15.04.3/src/CMakeLists.txt	2015-05-31 07:00:53.000000000 -0500
+++ src/cantor-15.04.3/src/CMakeLists.txt	2015-09-16 18:11:53.922524600 -0500
@@ -86,7 +86,7 @@ ki18n_wrap_ui(cantor_PART_SRCS extendeds
 configure_file (config-cantor.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/config-cantor.h )
 
 add_library(cantorpart MODULE ${cantor_PART_SRCS})
-set_target_properties(cantorpart PROPERTIES PREFIX "${CMAKE_SHARED_LIBRARY_PREFIX}")
+set_target_properties(cantorpart PROPERTIES PREFIX "lib")
 
 target_link_libraries(cantorpart KF5::Parts KF5::NewStuff
                      KF5::TextEditor ${Qt5XmlPatterns_LIBRARIES}
